src/components/PlayerProfile.vue

```diff
@@ -20,25 +18,23 @@
                 loading="Loading Avatar..."
               >
               </v-img>
-              <h3 class="justify-center"
-                style="
-                text-align:center;
-                margin-top:10px"
-                v-if="player.connectedTo!=null"
-                >
-                This User Is Online On {{player.connectedTo.name}}.
```

Comment:
Why are we dropping this completely? I think we should still show if the player is online. Perhaps even with a small visual indicator, like a green circle.